development/extensions/new_in_rhea.rst

```diff
@@ -98,15 +98,13 @@ Fortunately, the integration is pretty seamless and most existing extensions tha
   * ``generate_text_for_edit()``
   * ``generate_text_for_storage()``
   * ``strip_bbcode()``
-  * ``smiley_text()``
```

Comment:
Why can't it? I'm using smiley_text() in 3.2 the same as in 3.1 (to unrender smileys back to text) and it seems to work.